package.json

```diff
@@ -24,6 +24,7 @@
         "start": "node dist",
         "pkg:linux": "pkg . -t node18-linux-x64 -o bundle/dcli-linux",
         "pkg:macos": "pkg . -t node18-macos-x64 -o bundle/dcli-macos",
+        "pkg:linux-arm": "pkg . -t node18-linux-arm64 -o bundle/dcli-linux-arm",
```

Comment:
I was unsure if I should have added `pkg:linux-arm` to the `pkg` script. Let me know if it is the case

Reply:
Also, I did not include the docker build phase in the scripts. Can we think about this in a 2d MR, targeting the github workflows?